api-interop-layer/data/alerts/backgroundUpdateTask.js

```diff
@@ -146,14 +145,23 @@ export const updateAlerts = async ({ parent = parentPort } = {}) => {
 
     alert.event = rawAlert.properties.event;
 
-    const { locations, description } = parseLocations(
-      rawAlert.properties.description,
-    );
+    try {
```

Comment:
Twice today while I've been debugging there have been alerts that threw exceptions when trying to parse their locations. I wasn't able to capture either case, but now that particular bit of logic is wrapped in a try/catch and it logs the alert ID, which we should be able to lookup later from [IA State](https://mesonet.agron.iastate.edu/GIS/model.phtml)